WordPress/gutenberg
# Framework: Replace element-closest with registered vendor script

Closes #7159

This pull request seeks to eliminate `element-closest` dependency and imports from individual packages, instead replacing with a standard polyfill approach via server-side script enqueue.

Potential future improvements include:

- Improved association between script handle as being a polyfill (avoid `wp_add_inline_script` per dependent)
- Avoid polyfill dependencies needing to be kept in sync with current state of codebase using / not using (maybe always load)

**Testing instructions:**

Verify there are no regressions in `Element#closest` behaviors, notably in IE11 where it is not supported by default. For example, verify that inserting a quote block and ArrowDown'ing through its citation and into the addition of a default block (verifies usage of `Element#closest` in WritingFlow's use of `isInSameBlock`).
